src/cnv/Dockerfile

```diff
@@ -64,8 +64,12 @@ RUN wget https://github.com/samtools/samtools/releases/download/${SAMTOOLS_VERSI
 
 ## install ug-cn.mops (branch: ug.master):
 RUN cd /opt && \
-    git clone --branch ug.master https://github.com/Ultimagen/cn.mops.git cn.mops && \
-    cd cn.mops
+    if [ -d "cn.mops/.git" ]; then \
+        cd cn.mops && git pull; \
+    else \
+        git clone --branch ug.master https://github.com/Ultimagen/cn.mops.git cn.mops; \
```

Comment:
Is it maybe more robust to check out specific commit or tag? I think it will also prevent this weird case when the repo changes and the docker does not

Reply:
I assured the case of pulling with the if-else statement. 
I think a hard coded commit-id is not a good scenario. one may not be aware this should be manually updated here.